app/models/accounting/quickbooks/transaction_creator.rb

```diff
@@ -47,8 +47,8 @@ def customer_reference(organization)
         Customer.new(organization: organization, qb_connection: qb_connection).reference
       end
 
-      def department_reference(loan_id)
-        division = Loan.find_by(id: loan_id).division
+      def department_reference(loan)
+        division = loan.division
```

Comment:
Please combine this line with the one below. This assignment is unnecessary now.